components/PageHeader.jsx

```diff
@@ -83,64 +33,30 @@ const PageHeader = ({
 
     const stickyClass = isSticky ? ' usda-page-header--sticky' : '';
 
-    const showToolbar = (
-        !isEmpty(fyProps) ||
-        !isEmpty(shareProps) ||
-        !isEmpty(downloadProps)
-    );
-
     return (
-        <main id={id} className={`usda-page__container${classNames ? ` ${classNames}` : ''}`}>
```

Comment:
Thank you for removing main tag here!

Reply:
Yes this was a bug I introduced last sprint